hadoop-hdds/docs/content/feature/S3-Multi-Tenancy-Access-Control.md

```diff
@@ -66,3 +67,17 @@ The Ranger Sync thread does the following:
 2. Checks if default tenant roles are out-of-sync (could be caused by OM crash during user assign/revoke operation). Overwrites them if this is the case.
 3. Performs all Ranger update (write) operations queued by Ozone tenant commands from the last sync, if any.
    - This implies there will be a delay before Ranger policies and roles are updated for any tenant write operations (tenant create/delete, tenant user assign/revoke/assignadmin/revokeadmin, etc.). 
+
+
+## Sharing a bucket
+
+By default only the bucket owners have full access to the buckets they created.
+So in order to share a bucket with other users, a cluster admin or tenant admin will needs to manually create a new Ozone policy in Ranger for that bucket.  
```

Comment:
Have we tested this process? Is the bucket policy going to conflict with the bucket policy being created as part of the tenant?

Reply:
Good question. AFAIK resource denial takes precedence over any other allow rules, as depicted in this flow chart: https://docs.cloudera.com/cdp-private-cloud-base/7.1.7/security-ranger-authorization/topics/security-ranger-tags-and-policy-evaluation.html

Reply:
We tested this and it works without conflict.